### ISSUE TITLE
Port Forwarding is broken since a long time "parse error: Invalid numeric literal at line 1, column 5"

### ISSUE THREAD

**Issue by m4r1k** [haugene/docker-transmission-openvpn#1578]
**Describe the problem**
Before PIA _nextgen_ and transmission-openvpn 3.x, the PIA port forwarding used to work just fine. It's now broken for at least a couple of months. Albeit, it can definitely be a problem of mine.

Any help understanding where the problem is, it's much appreciated.

**Add your docker run command**
Follows my docker composer
```
version: '3.8'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - '/mnt/media/Torrents/:/data'
            - '/etc/localtime:/etc/localtime:ro'
        environment:
            - OPENVPN_PROVIDER=PIA
            - OPENVPN_CONFIG=austria
            - OPENVPN_USERNAME=user
            - OPENVPN_PASSWORD=pass
            - LOCAL_NETWORK=192.168.178.0/24
        logging:
            driver: json-file
            options:
                max-size: 10m
        restart: unless-stopped
        ports:
            - '9091:9091'
            - '8888:8888'
        image: haugene/transmission-openvpn:latest
```

**Logs**
Follows the logs
```
# docker-compose up
Recreating transmission-openvpn_transmission-openvpn_1 ... done
Attaching to transmission-openvpn_transmission-openvpn_1
transmission-openvpn_1  | Starting container with revision: 42d00652323b7b7fbee1689a54219599f4795a04
transmission-openvpn_1  | Creating TUN device /dev/net/tun
transmission-openvpn_1  | Using OpenVPN provider: PIA
transmission-openvpn_1  | Provider PIA has a custom setup script, executing it
transmission-openvpn_1  | Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.dLiipd
transmission-openvpn_1  | Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
transmission-openvpn_1  | Starting OpenVPN using config austria.ovpn
transmission-openvpn_1  | Modifying /etc/openvpn/pia/austria.ovpn for best behaviour in this container
transmission-openvpn_1  | Setting OpenVPN credentials...
transmission-openvpn_1  | adding route to local network 192.168.178.0/24 via 172.19.0.1 dev eth0
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 OpenVPN 2.4.9 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Apr 20 2020
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 library versions: OpenSSL 1.1.1g  21 Apr 2020, LZO 2.10
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 CRL: loaded 1 CRLs from file [[INLINE]]
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 TCP/UDP: Preserving recently used remote address: [AF_INET]156.146.60.31:1198
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 UDP link local: (not bound)
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 UDP link remote: [AF_INET]156.146.60.31:1198
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
transmission-openvpn_1  | Tue Dec  1 10:43:39 2020 [vienna402] Peer Connection Initiated with [AF_INET]156.146.60.31:1198
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 OpenVPN ROUTE6: OpenVPN needs a gateway parameter for a --route-ipv6 option and no default was specified by either --route-ipv6-gateway or --ifconfig-ipv6 options
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 OpenVPN ROUTE: failed to parse/resolve route for host/network: 2000::/3
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 TUN/TAP device tun0 opened
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 /sbin/ip link set dev tun0 up mtu 1500
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 /sbin/ip addr add dev tun0 10.30.112.179/24 broadcast 10.30.112.255
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 /etc/openvpn/tunnelUp.sh tun0 1500 1553 10.30.112.179 255.255.255.0 init
transmission-openvpn_1  | Up script executed with tun0 1500 1553 10.30.112.179 255.255.255.0 init
transmission-openvpn_1  | Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.30.112.179
transmission-openvpn_1  | Updating Transmission settings.json with values from env variables
transmission-openvpn_1  | Using existing settings.json for Transmission /data/transmission-home/settings.json
transmission-openvpn_1  | Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.30.112.179
transmission-openvpn_1  | Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed
transmission-openvpn_1  | Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
transmission-openvpn_1  | Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
transmission-openvpn_1  | Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
transmission-openvpn_1  | sed'ing True to true
transmission-openvpn_1  |
transmission-openvpn_1  | -------------------------------------
transmission-openvpn_1  | Transmission will run as
transmission-openvpn_1  | -------------------------------------
transmission-openvpn_1  | User name:   root
transmission-openvpn_1  | User uid:    0
transmission-openvpn_1  | User gid:    0
transmission-openvpn_1  | -------------------------------------
transmission-openvpn_1  |
transmission-openvpn_1  | STARTING TRANSMISSION
transmission-openvpn_1  | Provider PIA has a script for automatic port forwarding. Will run it now.
transmission-openvpn_1  | If you want to disable this, set environment variable DISABLE_PORT_UPDATER=yes
transmission-openvpn_1  | Transmission startup script complete.
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 WARNING: OpenVPN was configured to add an IPv6 route over tun0. However, no IPv6 has been configured for this interface, therefore the route installation may fail or may not work as expected.
transmission-openvpn_1  | Tue Dec  1 10:43:40 2020 Initialization Sequence Completed
transmission-openvpn_1  | parse error: Invalid numeric literal at line 1, column 5
transmission-openvpn_1  | Failed to acquire new auth token
```

**Host system:**
Ubuntu 20.04.1 using docker from snap (19.03.11 - dd360c7)

**Comment by haugene** [comment]
Hopefully fixed by #1602. Pull the latest image and let me know how that works :crossed_fingers:

**Comment by m4r1k** [comment]
I confirm that the problem is solved. Thanks a lot!

```
Starting container with revision: a4d65774f855a04070766e53346661f48c76fa0e
Creating TUN device /dev/net/tun
Using OpenVPN provider: PIA
Provider PIA has a custom setup script, executing it
Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.iLkoNJ
Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
Starting OpenVPN using config de_frankfurt.ovpn
Modifying /etc/openvpn/pia/de_frankfurt.ovpn for best behaviour in this container
Setting OpenVPN credentials...
adding route to local network 192.168.178.0/24 via 172.19.0.1 dev eth0
Sun Jan  3 12:47:26 2021 OpenVPN 2.4.9 x86_64-alpine-linux-musl [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Apr 20 2020
Sun Jan  3 12:47:26 2021 library versions: OpenSSL 1.1.1g  21 Apr 2020, LZO 2.10
Sun Jan  3 12:47:26 2021 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Sun Jan  3 12:47:26 2021 CRL: loaded 1 CRLs from file [[INLINE]]
Sun Jan  3 12:47:26 2021 TCP/UDP: Preserving recently used remote address: [AF_INET]138.199.18.132:1198
Sun Jan  3 12:47:26 2021 UDP link local: (not bound)
Sun Jan  3 12:47:26 2021 UDP link remote: [AF_INET]138.199.18.132:1198
Sun Jan  3 12:47:26 2021 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Sun Jan  3 12:47:26 2021 [frankfurt407] Peer Connection Initiated with [AF_INET]138.199.18.132:1198
Sun Jan  3 12:47:27 2021 OpenVPN ROUTE6: OpenVPN needs a gateway parameter for a --route-ipv6 option and no default was specified by either --route-ipv6-gateway or --ifconfig-ipv6 options
Sun Jan  3 12:47:27 2021 OpenVPN ROUTE: failed to parse/resolve route for host/network: 2000::/3
Sun Jan  3 12:47:27 2021 TUN/TAP device tun0 opened
Sun Jan  3 12:47:27 2021 /sbin/ip link set dev tun0 up mtu 1500
Sun Jan  3 12:47:27 2021 /sbin/ip addr add dev tun0 10.3.112.252/24 broadcast 10.3.112.255
Sun Jan  3 12:47:27 2021 /etc/openvpn/tunnelUp.sh tun0 1500 1553 10.3.112.252 255.255.255.0 init
Up script executed with tun0 1500 1553 10.3.112.252 255.255.255.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.3.112.252
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.3.112.252
Overriding blocklist-enabled because TRANSMISSION_BLOCKLIST_ENABLED is set to false
Overriding cache-size-mb because TRANSMISSION_CACHE_SIZE_MB is set to 32
Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed
Overriding download-queue-enabled because TRANSMISSION_DOWNLOAD_QUEUE_ENABLED is set to false
Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
Overriding peer-limit-global because TRANSMISSION_PEER_LIMIT_GLOBAL is set to 5000
Overriding peer-limit-per-torrent because TRANSMISSION_PEER_LIMIT_PER_TORRENT is set to 250
Overriding queue-stalled-enabled because TRANSMISSION_QUEUE_STALLED_ENABLED is set to false
Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
Overriding seed-queue-enabled because TRANSMISSION_SEED_QUEUE_ENABLED is set to false
Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
sed'ing True to true

-------------------------------------
Transmission will run as
-------------------------------------
User name:   root
User uid:    0
User gid:    0
-------------------------------------

STARTING TRANSMISSION
Provider PIA has a script for automatic port forwarding. Will run it now.
If you want to disable this, set environment variable DISABLE_PORT_UPDATER=true
Transmission startup script complete.
STARTING TINYPROXY
Found config file /etc/tinyproxy/tinyproxy.conf, updating settings.
Setting tinyproxy port to 8888
tinyproxy is on the PATH, run it
WARNING: logging deactivated (can't log to stdout when daemonized)
Tinyproxy startup script complete.
Sun Jan  3 12:47:27 2021 WARNING: OpenVPN was configured to add an IPv6 route over tun0. However, no IPv6 has been configured for this interface, therefore the route installation may fail or may not work as expected.
Sun Jan  3 12:47:27 2021 Initialization Sequence Completed
Running functions for token based port fowarding
Reserved Port: 26154  Sun Jan  3 12:47:33 UTC 2021
transmission auth not required
waiting for transmission to become responsive
transmission became responsive
setting transmission port to 26154
localhost:9091/transmission/rpc/ responded: "success"
Checking port...
Port is open: Yes
#######################
        SUCCESS
#######################
Port: 26154
Expiration Sun Mar  7 00:47:20 UTC 2021
#######################
Entering infinite while loop
Every 15 minutes, check port status
```
